### PR TITLE
Add MT42822 BlueField-2 integrated ConnectX-6 Dx in NicIdMap

### PR DESCRIPTION
**Pull request by zshi-redhat**
Signed-off-by: Zenghui Shi <zshi@redhat.com>

### CODE REVIEW

**Review by adrianchiris — APPROVED** [review]

File: api/v1/helper.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ var NicIdMap = []string{
 	"15b3 1017 1018", // ConnectX-5, PCIe 3.0
 	"15b3 101b 101c", // ConnectX-6
 	"15b3 1013 1014", // ConnectX-4
+	"15b3 a2d3 101e", // MT42822 BlueField-2 integrated ConnectX-6 Dx
```
Comment (adrianchiris) [inline api/v1/helper.go:50]: should be : "15b3 a2d6 101e"

Comment (zshi-redhat) [reply api/v1/helper.go]: Done

**Pull request merged**